src/app/components/ProposalInfo/index.tsx

```diff
@@ -2,97 +2,93 @@ import React from 'react';
 
 import dayjs from 'dayjs';
 import duration from 'dayjs/plugin/duration';
+import { Flex } from 'rebass/styled-components';
 import styled from 'styled-components';
 
-import { theme, Typography } from 'app/theme';
+import { Typography } from 'app/theme';
 import { ReactComponent as CalendarIcon } from 'assets/icons/calendar.svg';
 import { ReactComponent as PieChartIcon } from 'assets/icons/pie-chart.svg';
 import { ReactComponent as UserIcon } from 'assets/icons/users.svg';
 import { normalizeContent } from 'utils';
 
 dayjs.extend(duration);
 
-export default function ProposalInfo(props) {
-  const ProposalWrapper = styled.div`
-    border-radius: 10px;
-    flex: 1;
-    width: 100%;
-    background: #144a68;
-    border: 2px solid #144a68;
-    padding: 20px 25px;
-    transition: border 0.3s ease;
-    outline: none;
-    overflow: visible;
-    margin: 0 0 25px;
-    cursor: pointer;
+const ProposalWrapper = styled.div`
+  border-radius: 10px;
+  flex: 1;
+  width: 100%;
+  background: #144a68;
+  border: 2px solid #144a68;
+  padding: 20px 25px;
+  transition: border 0.3s ease;
+  outline: none;
+  overflow: visible;
+  margin: 0 0 25px;
+  cursor: pointer;
 
-    :hover,
-    :focus {
-      border: 2px solid #2ca9b7;
-    }
-  `;
-  const ApprovalSwatch = styled.div`
-    margin-right: 7px;
-    background: #2ca9b7;
-    height: 20px;
-    width: 20px;
-    border-radius: 5px;
-  `;
-  const RejectionSwatch = styled.div`
-    margin-right: 7px;
-    background: #fb6a6a;
-    height: 20px;
-    width: 20px;
-    border-radius: 5px;
-  `;
-  const Divider = styled.div`
-    border-bottom: 1px solid rgba(255, 255, 255, 0.15);
-    margin-bottom: 15px;
-  `;
-  const ContentText = styled(Typography)`
-    color: ${({ theme }) => theme.colors.text2};
-    margin-bottom: 15px;
-    font-size: 16px;
-  `;
+  :hover,
+  :focus {
+    border: 2px solid #2ca9b7;
+  }
+`;
+const ApprovalSwatch = styled.div`
+  margin-right: 7px;
+  background: #2ca9b7;
+  height: 20px;
+  width: 20px;
+  border-radius: 5px;
+`;
+const RejectionSwatch = styled.div`
+  margin-right: 7px;
+  background: #fb6a6a;
+  height: 20px;
+  width: 20px;
+  border-radius: 5px;
+`;
+const Divider = styled.div`
+  border-bottom: 1px solid rgba(255, 255, 255, 0.15);
+  margin-bottom: 15px;
+`;
+const ContentText = styled(Typography)`
+  color: ${({ theme }) => theme.colors.text2};
+  margin-bottom: 15px;
+  font-size: 16px;
+`;
 
+export default function ProposalInfo(props) {
   const { title, content, metadata } = props;
   const { voted, voters, approvePercentage, rejectPercentage, timestamp } = metadata;
-  const themes = theme();
   return (
     <ProposalWrapper>
       <Typography variant="h3" mb="10px">
         {title}
       </Typography>
       <ContentText>{normalizeContent(content)}</ContentText>
       <Divider />
-      <span style={{ display: 'flex', flexDirection: 'row' }}>
+      <Flex as="span">
```

Comment:
Why do you need to make the component as `span`? I think it is useless.

Reply:
<img width="980" alt="Screenshot 2021-07-17 at 2 52 18 PM" src="https://user-images.githubusercontent.com/87362499/126028731-2bd7e12b-c309-4e2b-9f0c-06505376fad0.png">

This is needed or it would look like this.

Reply:
https://rebassjs.org/flex/
If we use the `Flex` component as a container. you can control the alignment.